tavla/app/error.tsx

```diff
@@ -0,0 +1,40 @@
+'use client'
+
+import { Button } from '@entur/button'
+import { Heading3 } from '@entur/typography'
+import Image from 'next/image'
+import Link from 'next/link'
+import { useEffect } from 'react'
+import { logger } from 'utils/logger'
+import BeaverIllustration from 'assets/illustrations/Beaver.png'
+
+const log = logger.child({ module: 'appLevelErrorHandler' })
+export default function Error({
+    error,
+}: {
+    error: Error & { digest?: string }
+}) {
+    useEffect(() => {
+        log.error({
+            cause: error.cause,
+            stacktrace: error.stack,
+            message: error.message,
+            url: window.location.href,
+        })
```

Comment:
Skal vi legge til at vi logger `digest` også? Om en error ser på server-siden, så vil denne hjelpe oss med å finne ut va den eksakte feilmeldingen var: https://nextjs.org/docs/app/building-your-application/routing/error-handling#handling-server-errors og https://joulev.dev/blogs/throwing-expected-errors-in-react-server-actions